openquake/hazardlib/geo/surface/simple_fault.py

```diff
@@ -171,17 +172,22 @@ def from_fault_data(cls, fault_trace, upper_seismogenic_depth,
 
     @classmethod
     def get_fault_vertices_3d(cls, fault_trace, upper_seismogenic_depth,
-                              lower_seismogenic_depth, dip):
+                              lower_seismogenic_depth, dip, index_patch=None):
         """
-        Get surface main vertexes.
-
+        Get surface main vertices.
         Parameters are the same as for :meth:`from_fault_data`, excluding
         mesh spacing.
 
+        :param index_patch:
```

Comment:
I would write:
Index of the fault-patch that will be returned by this method. The fault patch numbering follows the same logic of the right-hand rule i.e. patch with index 0 is the first along the trace.